<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>388</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Academy Engraved LET</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>天堂m-自動轉日記</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="left_btn1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>標楷體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>日記左格子1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="left_btn2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>標楷體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>日記左格子2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>111</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>標楷體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>模擬器類型</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_vmtype">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>標楷體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>161</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>標楷體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>模擬器畫面尺寸</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_vmsize">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>標楷體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QGraphicsView" name="left_ghView1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="left_ghView2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="start_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>標楷體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>開始自動轉日記</string>
    </property>
   </widget>
   <widget class="QPushButton" name="end_btn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>標楷體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退出程式</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>標楷體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>自動取得日記本物品次數 :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="right_btn1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>標楷體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>日記右格子1</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="right_ghView1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>110</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="right_btn2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>標楷體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>日記右格子2</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="right_ghView2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>170</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="hs_diarycount">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>249</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lab_displaydiraycount">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>243</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>標楷體</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_sysmsg">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>標楷體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>系統訊息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setdevice_btn">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>標楷體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>set device</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_sysmsg">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>50</y>
      <width>391</width>
      <height>291</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>細明體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_device">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>細明體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="getdevice_btn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>標楷體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>get device</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearmsg_btn">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>17</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>標楷體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>清除訊息內容</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>877</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
